### DISCUSSION TITLE
Holonomic mode is on, but state.GetPoseMeters's rotation uses the heading instead of the rotation we specified

### DISCUSSION THREAD

**Discussion by SteptimusHeap** [mjansen4857/pathplanner#240] [General]
I'm pushing the state's pose's rotation to smartdashboard, and instead of matching the rotation of the actual robot, is matches the heading of the robot, even though I have holonomic mode on in pathplanner

Her's my code: https://github.com/Bedford-Express-1023/2023Project-0.1/tree/Drivetrain
Here's my path: https://imgur.com/a/Lh55lp7

**Comment by mjansen4857** [comment]
This is working as expected. The rotation in the pose needs to be the heading in order to work correctly with path following commands. This is why holonomic rotation is a separate field.
